_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -0,0 +1,83 @@
+
+## Succeeding own IdentityAttributes
+
+When talking about [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute), we need to distinguish three cases:
+
+- The Identity maintains an Attribute about itself. It is stored in the `content` field of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute), whose `shareInfo` is undefined. Since this LocalAttribute is created for the Identity's private repository of Attributes, it is referred to as **RepositoryAttribute**.
+- When sharing a RepositoryAttribute, a copy of the IdentityAttribute is created and stored in the `content` field of a LocalAttribute with a defined `shareInfo`. We call this LocalAttribute an **own shared IdentityAttribute**.
+- Receiving a shared IdentityAttribute from a peer leads to the creation a LocalAttribute with according `content` and a defined `shareInfo` for the recipient. We call this LocalAttribute a **peer shared IdentityAttribute**.
+
+Discussing the succession of IdentityAttributes requires some background knowledge about this differentiation and the behavior of shared Attributes.
+Hence, we will look at the process of creating, sharing and succeeding an IdentityAttribute step by step.
+
+### Creating a RepositoryAttribute
+
+Enmeshed allows you to store data about yourself in the form of IdentityAttributes.
+When [creating an IdentityAttribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}), it is not necessary to share it immediately with a peer.
+Instead, it will be stored in the `content` property of a LocalAttribute with an undefined `shareInfo`.
+We refer to these unshared LocalAttributes as RepositoryAttributes, since they make up your private repository of Attributes.
+
+In the following examples, the `createdAt` property of the LocalAttribute is omitted, since it isn't required for the explanation.
+{: .notice--info}
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/33654502-420f-41bd-801a-e0d4c1564df0" id="zs0-w2Ag-nJ4"></iframe></div>
+
+### Sharing a RepositoryAttribute
+
+Now, in order to [share a RepositoryAttribute]({% link _docs_use-cases/use-case-consumption-share-an-identityattribute.md %}), you need to send an according [Request]({% link _docs_integrate/data-model-overview.md %}#request) to the peer you want to share it with.
+If they [accept your Request]({% link _docs_integrate/share-own-attribute-to-peer.md %}), a new LocalAttribute will be created at the peer's side.
+This peer shared IdentityAttribute has the same `content` like your RepositoryAttribute and, in addition, a defined `shareInfo` property.
+It stores the address of the `peer` who shared the Attribute with them, i.e. your address, and a reference to the Request that was used to share the Attribute.
```

Comment:
We normally capitalise "Address" in this context (e.g. compare with https://enmeshed.eu/integrate/data-model-overview#localattributeshareinfo).

Same for other locations